test/Stats.unittest.js

```diff
@@ -174,6 +180,16 @@ describe(
 					warnings: []
 				});
 			});
+
+			it('Given showErrors it should return error with its stack', () => {
+				const fakeError = new Error('FAKE STACK');
+				fakeCompilation.errors = [fakeError];
+				const mockStats = new Stats(fakeCompilation);
+
+				const result = mockStats.toJson({ errorDetails: true });
+				
+				expect(result.errors[0]).toEqual(`${fakeError.message}\n${fakeError.stack}`);
```

Comment:
```suggestion
				expect(result.errors[0]).toEqual(fakeError.stack);
```

stack already include the message.

Reply:
LGTM